packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -59,23 +266,28 @@ def _create_file(
     "--class-name",
     help="Name for the Python Measurement Plug-In Client Class in the generated module.",
 )
-@click.option(
-    "-a",
-    "--all",
-    is_flag=True,
-    help="Creates Python Measurement Plug-In Client for all the registered measurement services.",
-)
 @click.option(
     "-o",
     "--directory-out",
     help="Output directory for Measurement Plug-In Client files. Default: '<current_directory>/<module_name>'",
 )
+@click.option(
+    "-i",
+    "--interactive",
+    is_flag=True,
+    help=(
+        "Enables interactive mode for creating Measurement Plug-In Client files. "
+        "If no parameters are provided, this mode will be activated by default."
```

Comment:
Do we need to make the `-i` command as optional? IMO, I would recommend throwing an error if the user does not provide any subcommand along with the `ni-measurement-plugin-client-generator` command.

Reply:
Previously in chat:
> One other approach that some command line utilities use is to enter interactive mode when no arguments or options are specified. Tools like nslookup, netsh (Windows), lldpcli (Linux), etc. do this.

FYI @MounikaBattu17 This suggestion was really an alternative to an explicit `-i` option. Using an explicit `-i` option and returning an error if no options are specified is fine with me.

Reply:
Sure! Thanks for the confirmation. With the usage of click-option-group, we will be throwing error if required option (-a/-i/-s) is not specified.